### PR TITLE
added new node globals, setImmediate and clearImmediate to whitelist

### PR DESCRIPTION
**Pull request by vicapow**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -48,6 +48,14 @@ exports['clearInterval is a default'] = function () {
...
   assert.ok(~g.whitelist.indexOf(clearInterval));
 };
+exports['setImmediate is a default'] = function () {
+  var g = gleak();
+  assert.ok(~g.whitelist.indexOf(setImmediate));
+};
+exports['clearImmediate is a default'] = function () {
+  var g = gleak();
+  assert.ok(~g.whitelist.indexOf(clearImmediate));
+};
```
Comment (aheckmann) [inline test/index.js:58]: lets only test these if using node >= 0.10, see https://github.com/aheckmann/gleak/blob/master/test/index.js#L9

File: index.js [hunk 2]
```diff
@@ -84,6 +84,8 @@ Gleak.prototype.whitelist = [
   , setInterval
   , clearTimeout
   , clearInterval
+  , setImmediate
+  , clearImmediate
```
Comment (aheckmann) [inline index.js]: we should only add these if running >= 0.10, see: https://github.com/aheckmann/gleak/blob/master/index.js#L114

**Comment by vicapow** [comment]
i tried to fix the unit tests too, but i couldn't figure out what was wrong. has something to do with express 2.0 -> 3.0 and calling `.address()`

**Comment by aheckmann** [comment]
0.3.0 is out with the fixes.

**Comment by aheckmann** [comment]
@vicapow express v3 apps are not http servers. you get access to it when calling `.listen()`.

**Pull request closed without merging**